src/style.css

```diff
@@ -0,0 +1,152 @@
+:root {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+/* BLOCK "CARD" */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  padding: 16px 3px 40px;
+  height: 134px;
+}
+
+.card__product-name {
+  padding-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__product-code {
+  padding-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
```

Comment:
move all repeated colors into css variables

Reply:
done

to simplify color management, I replaced all colors with vars